widgets/hero/hero.php

```diff
@@ -477,6 +477,18 @@ function wrapper_data_filter( $data, $instance ) {
 		}
 		return $data;
 	}
+
+  function modify_instance( $instance ) {
+		if ( !empty( $instance['controls']['nav_always_show_mobile'] ) && $instance['controls']['nav_always_show_mobile'] ) {
+			$global_settings = $this->get_global_settings();
+
+			if ( ! empty( $global_settings['responsive_breakpoint'] ) ) {
+				$instance['controls']['breakpoint'] = $global_settings['responsive_breakpoint'];
```

Comment:
There are a few problems with this:
* This exact same code is being duplicated in each of the widgets that extend the base slider widget class.
* It's a bit hacky and obscure to create a field on the `$instance` which might look like it comes from the controls section of the widget form, but doesn't. This would likely not be necessary if the duplicated code was in the base slider class.

You could perhaps retrieve the global setting for the breakpoint in the base slider's `slider_settings` function. No need to pass it through from the widget.

Reply:
That... that makes a lot more sense. Done.